### PR TITLE
(FEAT) Simple file replacement of ogg samples (Replace microphone (de)activation noises that are made with generated noise)

### PR DESCRIPTION
**Pull request by hvanmegen**
TYPE(Scope): Replace sound samples ( https://github.com/mumble-voip/mumble/issues/5891 )

Did two things:
- Renamed original microphone (de)activation sounds by adding `_sweep` to the name.
- Added new samples (handcrafted in Audacity) based on noise (much like Teamspeak's (de)activation sound effects called `on_noise.ogg` and `off_noise.ogg`
- Made the necessary changes in the list of samples under src/mumble/mumble.qrc

That is all, thank you for considering these new samples.

### CODE REVIEW

**Comment by Hartmnt** [comment]
@Krzmbrzl @davidebeatrici Is there consensus on changing the sound on and off cues to noise instead of sweep? I personally do not care and I do not use on/off cues.

**Comment by Krzmbrzl** [comment]
I don't mind - haven't listened to the new sounds yet but as long as users can revert back to the old ones if they don't like the new ones, I don't see an issue in just changing them and then see what the community has to say about the change.

**Comment by Hartmnt** [comment]
@hvanmegen Sorry that this took so long. Are you still willing to work on this?

If so the following needs to be done:
* Change the default setting for the on/off cue to the name of the new files you added here
* Change the default of the mute cue to ``off_sweep.ogg`` (because I feel like the mute cue should be different from the regular off cue)
* Make sure that there is an upgrade path for clients that have ``on.ogg`` and ``off.ogg`` configured such that they will automatically use ``on_noise.ogg`` and ``off_noise.ogg``
* Squash your commits into a single commit and improve commit title and description

**Comment by hvanmegen** [comment]
I'm still willing to work on it.. I simply lost interest to make the change after I couldn't figure out what I needed to change to get this done and life things got in the way (quarantine at home was no longer a thing & I changed jobs, no longer needing VoIP tools as much).

I still think this would be a great feature as being aware of microphone activation is one of the defining advantages that TeamSpeak has for me over Discord.. since I still want to move away from TeamSpeak to Mumble, I would very mucht still like these files to become the default microphone threshold activation and deactivation sounds for Mumble, so I can transition as many people from TeamSpeak to Mumble as I can, but I'm also aware that some people might prefer the older version of the sounds, so some way of adding both would be needed, and not simply replacing the files.

--

TL;DR: I'll do my best to find the time to redo the samples and add a third one for microphone muting (complete with a reproduction path for anyone that wants to recreate them) and then I'll try to do the things you suggested in your previous comment.

**Comment by hvanmegen** [comment]
I'll close this for now and reopen it with the new ones.

**Comment by Hartmnt** [comment]
> microphone muting

I was specifically talking about the "mute cue" which is a sound that gets played while you are muted to inform you that no speech is being sent. I feel like the original "off cue" (the bleep sound) is good enough for that so you do not have to re-create that one.

**Comment by hvanmegen** [comment]
> > microphone muting
> 
> I was specifically talking about the "mute cue" which is a sound that gets played while you are muted to inform you that no speech is being sent. I feel like the original "off cue" (the bleep sound) is good enough for that so you do not have to re-create that one.

Ok, I'll focus on just the microphone-on and microphone-off sounds then.. I haven't used the thing, but yeah, a simple bleep thing is fine. It's just that the microphone response sounds really need to be more subtle in order to be useful.
